src/task_queue/database_tasks/media_library_cleanup.py

```diff
@@ -33,7 +33,10 @@ def remove_duplicate_images(task):
              print("Generating hashes")
              result = media_store.generate_hashes(None, None, None)
 
-        communities = flag.enabled_communities()
+        if flag and do_updates: 
+            communities = flag.enabled_communities()
```

Comment:
I think if the flag is enabled, but no communities selected, this will produce an empty queryset and it will not do any reporting on duplicates, but probably blow up.